dirac.cfg

```diff
@@ -424,19 +424,33 @@ Systems
 }
 Resources
 {
+  # Section for identity providers, subsections is the names of the identity providers
+  # https://dirac.readthedocs.org/en/latest/AdministratorGuide/Resources/identityprovider.html
   IdProviders
   {
-    CheckIn
+    ## EGI Checkin type:
+    EGI Checkin
```

Comment:
I would not suggest to have spaces

Reply:
The reason why there are spaces here is that the name of this section is the name of IdP, which is IMHO convenient. I tested this "spaced section name" for authorization and it seems to work for me.

Reply:
It looks "nice" but I would still not have spaces, they could be problematic in certain corner cases. "EGI_CheckIn" is too ugly?

Reply:
>"EGI_CheckIn" is too ugly?

Everything is relative, as for me it is good if this variable is used out of sight of the user. If these names are part of the UI, then IMHO it should be a readable name.

But I agree that I can place `replace("_", "")` where is needed.